msteams-platform/tabs/how-to/create-channel-group-tab.md

```diff
@@ -1,23 +1,65 @@
 ---
-title: Create a channel tab or group tab
-description: Create custom channel, group tab with Node.js, ASP.NET Core, ASP.NET Core MVC. Generate app, create package, build and run app, secret tunnel, upload to Teams and build your first app using Blazor.
+title: Create a configurable tab
```

Comment:
We have two pages that are doing the same thing: `create a configurable tab` and `create a configurable page`. There should be two pages: `create a configurable tab` and `create a removal page`. Those are the only two concepts someone building a configurable tab would need to know in order to get started.